### PR TITLE
Change separator

### PR DESCRIPTION
**Pull request by devicedev**
https://etvas.atlassian.net/browse/ETV-8237

### CODE REVIEW

File: src/components/RuleBuilder/utils/export.js [hunk 1]
```diff
@@ -36,11 +36,11 @@ const exportRules = (rules, options) =>
         return parseInt(value, 10)
       }
 
-      const split = value.split('.')
-      return [parseInt(split?.shift(), 10), parseInt(split?.pop(), 10)]
+      const split = value.split(',')
+      return `${parseInt(split?.shift(), 10)},${parseInt(split?.pop(), 10)}`
```
Comment (hututudor) [inline src/components/RuleBuilder/utils/export.js:40]: Why is this splitting and formatting necessary? We already have the value in the `left,right` format, right?

Comment (devicedev) [reply src/components/RuleBuilder/utils/export.js]: 94f07e8e31ff891ff2542e8f5f6633ce0cf3b32c

**Review by hututudor — APPROVED** [review]

File: src/components/RuleBuilder/utils/export.js [hunk 2]
```diff
@@ -11,40 +11,25 @@ export const exportRuleBuilder = (
...
 
-const exportGroups = (groups, combinedRuleOptions, absoluteRuleOptions) =>
+const exportGroups = (groups, absoluteRuleOptions) =>
   groups.map(({ not, absolute, combined, combinator, advancedTargeting }) => ({
     not,
     combinator,
-    combined: exportRules(combined, combinedRuleOptions),
+    combined: exportRules(combined),
     absolute: advancedTargeting
       ? exportRules(absolute, absoluteRuleOptions)
```
Comment (hututudor) [inline src/components/RuleBuilder/utils/export.js:20]: Extra argument for `exportRules`

Comment (hututudor) [reply src/components/RuleBuilder/utils/export.js:20]: On line 20

Comment (devicedev) [reply src/components/RuleBuilder/utils/export.js]: 9abaf5f6eb3fa71d22afef8c09c7ea513526e2e8

**Pull request merged**